tests/src/test/java/features/backendfeatures/1-generateUrlAndSubmissionForExistingGuid.feature

```diff
@@ -46,16 +46,7 @@ Feature: When user uploads the documents, id is authenticated user details, navi
     ## Call to get document wth filename ##
     Given "SUBMISSION" id with filename path is submitted with GET http request
     Then Verify status code is 200 and content type is not json
-
- # @backend
-  Scenario: Verify if cso-bambora relation can be set and payment process service can be created
-     ## Call to upload the document ##
-    Given POST http request is made to "DOCUMENT_SUBMISSION" with valid existing CSO account guid and a single pdf file
-    When status code is 200 and content type is verified
-    Then verify submission id and document count is received
-    ## Call to generate url ##
-    Given POST http request is made to "GENERATE_URL_API" with client application, court details and navigation urls
-    When status code is 200 and content type is verified
-    Then verify expiry date and eFiling url are returned with the CSO account guid and submission id
-    ## call to set bambora ##
-
+      ## Call to submit document ##
```

Comment:
if this is commented and you don't need it, remove it please.

Reply:
This is not a commented out but an actual "comment" syntax in gherkin.